sysbuild/Kconfig.mcuboot

```
@@ -120,6 +120,11 @@ endchoice
 
 endif
 
+config FPROTECT_ALLOW_COMBINED_REGIONS
+	bool
+	default y
+	depends on (BOOTLOADER_MCUBOOT && !SECURE_BOOT_APPCORE)
```

```suggestion
	depends on SOC_SERIES_NRF54LX && !SECURE_BOOT_APPCORE
```
move the above `endif` to below this so it's included in the if check